### ISSUE TITLE
SATP (epic) - visualization and inspection of cross chain transactions

### ISSUE THREAD

**Issue by RafaelAPB** [hyperledger-cacti/cacti#3111]
Part 1: refactor CBDC example UI (if needed)

Part 2:  visualization tools for views
A blockchain view might be a difficult concept to comprehend. Create visualization tools to view the different parts of the blockchain view, including segmentation by chain and gateway consortium. Some inspiration:

https://layerzeroscan.com/
https://tx.town/v/eth-btc
https://axelarscan.io/

This issue aims to create a SATP Admin UI to inspect the cross-chain transactions and views. The UI essentially has to call the BUNGEE endpoints and other SATP endpoints.

Part 3: Create a Grafana dashboard focusing on metrics (eg performance) to complement the created UI

**Comment by eduv09** [comment]
@RafaelAPB please assign me this one

**Comment by RafaelAPB** [comment]
Depends on #3110

**Comment by RafaelAPB** [comment]
Issue updated

**Comment by outSH** [comment]
@RafaelAPB Have you considered adding this UI as an app inside https://github.com/hyperledger-cacti/cacti/tree/main/packages/cacti-ledger-browser? I wanted to work on this some time ago and prepare some PoC, but other projects got in a way.. I would love to revisit the idea some time in the future, sounds like a good candidate for mentorship project as well

**Comment by RafaelAPB** [comment]
Yes! Per our conversation, we would like to include any UI efforts in this package. We ought to think it this is a more generic package (like an admin interface for cacti/satp) or more concrete (in the ledger browser)

**Comment by outSH** [comment]
`ledger-browser` is kinda unfortunate name, because in it's current form is not only limited to reading ledger state, but can also be used to develop apps talking directly to the components (for admin / interaction purposes).

Please keep me in the loop about this issue, and I'll try to revisit the idea once I get a spare time :)

**Comment by RafaelAPB** [comment]
this is done. we should explore synergies with ledger-browser

This issue will be considered if the depending package remains, as part of the cleanup initiative: https://github.com/hyperledger-cacti/cacti/issues/4037
